Profile Folder/chrome/locale/tr-TR/dtd/gm.dtd

```diff
@@ -1,4 +1,4 @@
 <!ENTITY gm.openInTabNext		"Sonraki sefer sihirbazı yeni sekmede aç">
-<!ENTITY gm.clickSoundFeedback	"[!!! Éñáβℓè çℓïçƙ ƨôúñδ ƒèèδβáçƙ !!!]">
+<!ENTITY gm.clickSoundFeedback	"Tıklama için ses geri bildirimini etkinleştir">
```

Comment:
Would it be possible to remove "Enable" from `gm.clickSoundFeedback` and `gsettings.enableForceColorizeAddressBar`? The toggle already implies that it's for enabling/turning on, so we only need the setting name.

I will also rename `gsettings.enableForceColorizeAddressBar` to `gsettings.forceColorizeAddressBar` in the future to avoid confusion.